### DISCUSSION TITLE
Need help with Empiretown..

### DISCUSSION THREAD

**Discussion by Lukascc** [community/community#59698] [Programming Help]
### Select Topic Area

Question

### Body

hey i installed latest artifact v6537 and [MAIN] but when i start the server it spits out lots of errors, and when i connect i can't go past registration because of the errors. there's some of them: 
   script:esx-radio] SCRIPT ERROR: @esx-radio/server.lua:2: No such export addChannelCheck in resource pma-voice

[   script:bpt_wallet] SCRIPT ERROR: citizen:/scripting/lua/scheduler.lua:90: ox_lib should be started before this resource.
[   script:bpt_wallet] > fn (@ox_lib/init.lua:13)
[   script:bpt_wallet] SCRIPT ERROR: @bpt_wallet/server/server.lua:28: attempt to index a nil value (global 'lib')
[   script:bpt_wallet] > fn (@bpt_wallet/server/server.lua:28)

**Comment by amianicaroline** [comment]
hi i would like you to help me to the answer to GitHub plz

**Comment by Lukascc** [comment]
Ok

2023-06-30, pn 13:23, amianicaroline ***@***.***> rašė:

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Lukascc ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by bitpredator** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> hey i installed latest artifact v6537 and [MAIN] but when i start the server it spits out lots of errors, and when i connect i can't go past registration because of the errors. there's some of them: script:esx-radio] SCRIPT ERROR: @esx-radio/server.lua:2: No such export addChannelCheck in resource pma-voice
> 
> [ script:bpt_wallet] SCRIPT ERROR: citizen:/scripting/lua/scheduler.lua:90: ox_lib should be started before this resource. [ script:bpt_wallet] > fn (@ox_lib/init.lua:13) [ script:bpt_wallet] SCRIPT ERROR: @bpt_wallet/server/server.lua:28: attempt to index a nil value (global 'lib') [ script:bpt_wallet] > fn (@bpt_wallet/server/server.lua:28)


Hi @Lukascc , you need to edit your server.cfg and run the [ox] package first instead of the pack containing bpt_wallet, I've been fixing these problems for some time now, I assume you're using my "Empiretown" repository.
Greetings bitpredator
